packages/core/src/domain/tracekit/computeStackTrace.ts

```diff
@@ -61,6 +65,26 @@ function parseChromeLine(line: string): StackFrame | undefined {
   }
 }
 
+const CHROME_ANONYMOUS_FUNCTION_RE =
+  // eslint-disable-next-line local-rules/disallow-side-effects
+  new RegExp(`^\\s*at ?${url}${position}?${position}??\\s*$`, 'i')
+
+function parseChromeAnonymousLine(line: string): StackFrame | undefined {
+  const parts = CHROME_ANONYMOUS_FUNCTION_RE.exec(line)
+
+  if (!parts) {
+    return
+  }
+
+  return {
+    args: [],
+    column: parts[3] ? +parts[3] : undefined,
+    func: UNKNOWN_FUNCTION,
+    line: parts[2] ? +parts[2] : undefined,
+    url: parts[1] ?? undefined,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏the URL part isn't optional
```suggestion
    url: parts[1],
```